doc/doxygen/src/roadmap.md

```diff
@@ -9,85 +9,66 @@ The text and items below are tentative, up for discussion, to be updated by regu
 (contact/steering: [Martine](https://github.com/miri64))
 
 1. ICN stack support clean-up
-2. discuss mid- and long-term plans for network stack maintenance & development (GNRC vs other supported stacks)
-3. Support for CoAP blockwise transfer, observe. Also see [Ken's](https://github.com/kb2ma) unofficial [CoAP for RIOT](https://github.com/kb2ma/RIOT/wiki/CoAP-Home) page.
-4. revisit network time synchronization
+1. discuss mid- and long-term plans for network stack maintenance & development (GNRC vs other supported stacks)
+1. revisit network time synchronization
 
 
 
 # Network Stack Low layers
 (contact/steering: [Peter](https://github.com/PeterKietzmann))
```

Comment:
I think @jia200x is the better contact.

Reply:
Again, let's have a separate PR for this. I can open that PR once this is in.